tempesta_fw/t/unit/test_http_parser.c

```diff
@@ -146,37 +146,40 @@ do_split_and_parse(unsigned char *str, int type)
 #define FOR_REQ(str)						\
 	TEST_LOG("=== request: [%s]\n", str);			\
 	chunks = 1;						\
-	while(TRY_PARSE_EXPECT_PASS(str, FUZZ_REQ))
+	while (TRY_PARSE_EXPECT_PASS(str, FUZZ_REQ))
 
 #define EXPECT_BLOCK_REQ(str)					\
 do {								\
 	TEST_LOG("=== request: [%s]\n", str);			\
 	chunks = 1;						\
-	while(TRY_PARSE_EXPECT_BLOCK(str, FUZZ_REQ));		\
+	while (TRY_PARSE_EXPECT_BLOCK(str, FUZZ_REQ));		\
 } while (0)
 
 #define FOR_RESP(str)						\
 	TEST_LOG("=== response: [%s]\n", str);			\
 	chunks = 1;						\
-	while(TRY_PARSE_EXPECT_PASS(str, FUZZ_RESP))
+	while (TRY_PARSE_EXPECT_PASS(str, FUZZ_RESP))
 
 #define EXPECT_BLOCK_RESP(str)					\
 do {								\
 	TEST_LOG("=== response: [%s]\n", str);			\
 	chunks = 1;						\
-	while(TRY_PARSE_EXPECT_BLOCK(str, FUZZ_RESP));		\
+	while (TRY_PARSE_EXPECT_BLOCK(str, FUZZ_RESP));		\
 } while (0)
 
 TEST(http_parser, parses_req_method)
 {
-	FOR_REQ("GET / HTTP/1.1\r\n\r\n")
+	FOR_REQ("GET / HTTP/1.1\r\n\r\n") {
 		EXPECT_EQ(req->method, TFW_HTTP_METH_GET);
+	}
 
-	FOR_REQ("HEAD / HTTP/1.1\r\n\r\n")
+	FOR_REQ("HEAD / HTTP/1.1\r\n\r\n") {
```

Comment:
Why do we need curly brackets here and in similar statement below? All macros like `EXPECT_EQ()` are implemented correctly with `do {} while (0)` where necessary. In all practical terms, it's a single statement that doesn't require brackets.